cv32e40s/docs/VerifPlans/Simulation/privileged_spec/CV32E40SX_PMA_VerifPlan.csv

```diff
@@ -290,19 +287,21 @@ COV: TODO",TODO missing cover.
 In both cases, data_req_o should remain low for both operations.",Self Checking Test,Directed Self-Checking,Testcase,DTC: cv32e40s/tests/programs/custom/pma/,TODO test is commented out
 ,,,,,Check against RM,Constrained-Random,Functional Coverage,"RTC: cv32e40s/tests/programs/corev-dv/corev_rand_pma_test/
 
-COV: TODO",TODO missing cover.
+COV: *.pma_cov_data_i.gen_rvfi_cg.cover_item_covergroup_cg_rvfi_inst_rvfi_cg_cross_x_aligned_pmafault_loadstore_firstfail_*
+
+COV: *.pma_cov_data_i.gen_rvfi_cg.cover_item_covergroup_cg_rvfi_inst_rvfi_cg_cross_x_aligned_loadstore_boundary_bin_*",
 UM 0e447ac,Zce Extension,Push instructions,"push, push.e, c.push, c.push.e instructions shall only occur for main memory regions, any such *push* attempts to I/O shall cause a precise load access fault (exception code 5)","Attempt *push* operations to main memory and IO, verify that all *push* attempts to IO causes a precise store access fault (code 5) and that this does not occur for main memory.",Self Checking Test,Directed Self-Checking,Testcase,DTC: cv32e40s/tests/programs/custom/pma/,TODO test is commented out
 ,,,,,Check against RM,Constrained-Random,Functional Coverage,"RTC: TODO
 
-COV: TODO","TODO missing random test?
+COV: *.pma_cov_data_i.cover_item_covergroup_cg_inst_cg_inst_cross_x_loadstoreexec_main_pushpop_*","TODO missing random test?
 
-TODO missing cover."
-,,Pop instructions,"pop, popret, pop.e, popret.e, c.pop, c.popret, c.pop.e, c.popret.e shall only occur for main memory regions, any such *pop* attempts from I/O shall cause a precise load access fault (exception code 7)","Attempt *pop* operations to main memory and IO, verify that all *pop* attempts to IO causes a precise loac access fault (code 7) and that this does not occur for main memory.",Self Checking Test,Directed Self-Checking,Testcase,DTC: cv32e40s/tests/programs/custom/pma/,TODO test is commented out
+Note: Cover doesn't check that it goes through WB."
+,,Pop instructions,"pop, popret, pop.e, popret.e, c.pop, c.popret, c.pop.e, c.popret.e shall only occur for main memory regions, any such *pop* attempts from I/O shall cause a precise load access fault (exception code 7)","Attempt *pop* operations to main memory and IO, verify that all *pop* attempts to IO causes a precise loac access fault (code 7) and that this does not occur for main memory.",Self Checking Test,Directed Self-Checking,Testcase,DTC: cv32e40s/tests/programs/custom/pma/,TODO test is commented out.
 ,,,,,Check against RM,Constrained-Random,Functional Coverage,"RTC: TODO
 
-COV: TODO","TODO missing random test?
+COV: *.pma_cov_data_i.cover_item_covergroup_cg_inst_cg_inst_cross_x_loadstoreexec_main_pushpop_*","TODO missing random test?
 
-TODO missing cover."
+Note: Cover doesn't check that it goes through WB."
```

Comment:
These changes looks fine. 
But is there any plans concerning the notes you have added? Or is the plan to just mention it, and dont do any thing about it?

Reply:
> is the plan to just mention it, and dont do any thing about it?

Short answer: Yes.

Long answer:
1)
For notes like the one above, I deem it to be too detailed and pedantic to be practical.
There is still a chance later, during signoff, that we decide that these should get a higher priority, but for now I don't think there is much ROI on it.
2)
For the one coverage vplan item that I did not link to (relating to isacov), there is a separate ticket for bringing isacov up to date.
But I should not trust that I remember this about PMA later, so I just added a reminder comment on that ticket, thanks.